test/issues/338-test.js

```diff
@@ -0,0 +1,11 @@
+const Table = require('../..');
+
+test('closes href tag on truncated content', () => {
+  const href = 'http://example.com';
+
+  const table = new Table({ colWidths: [10], style: { border: [], head: [] } });
+
+  table.push([{ content: 'looooooooooooong', href }]);
+
+  expect(table.toString().includes('\x1B]8;;\x07')).toEqual(true);
```

Comment:
I didn't do this initially because the string-literal table doesn't look as nice as the ones in the other issue tests.  But I made the change and it seems fine.